spec/lib/efile/dependent_eligibility/qualifying_relative_spec.rb

```diff
@@ -4,7 +4,7 @@
   subject { described_class.new(dependent, TaxReturn.current_tax_year) }
 
   context 'with a totally qualifying relative' do
-    let(:dependent) { create :qualifying_relative }
+    let(:dependent) { create :qualifying_relative, intake: create(:ctc_intake) }
     let(:test_result) do
```

Comment:
Not related to this pull request, but on a skim, it's hard to tell if `test_result` is input to the tests, or the expected output. I now see it's the expected output, so I'd prefer a rename to `expected_test_result`. Not a blocker at all.